### PR TITLE
watch mkdir logs

### PR DESCRIPTION
**Pull request by sparkoo**
### What does this PR do?
watch mkdir pod logs when workspace started in debug

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/16161

#### Docs PR
will be part of UI Tasks

### CODE REVIEW

**Comment by sparkoo** [comment]
ci-build

**Review by mshaposhnik — APPROVED** [review]

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PVCSubPathHelper.java [hunk 1]
```diff
@@ -154,6 +186,17 @@ void execute(
       String pvcName,
       String[] commandBase,
       String... arguments) {
+    execute(null, workspaceId, namespace, pvcName, commandBase, Collections.emptyMap(), arguments);
```
Comment (amisevsk) [inline infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PVCSubPathHelper.java:189]: Is this `execute` function just for test cases? If so, doesn't it make sense to just remove it and make the actual function below visible for testing? It's strange to have what seems to be a wrapper for setting `identity=null`.

**Review by amisevsk — APPROVED** [review]
LGTM

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PVCSubPathHelper.java (hunk 1 above)
Comment (sparkoo) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PVCSubPathHelper.java:189]: no, it's called from `removeDirsAsync`, which is async operation where we don't have the identity.

**Pull request merged**